airbyte-webapp/src/components/ui/Table/SortableTableHeader/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { faCaretDown, faCaretUp } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React, { PropsWithChildren } from "react";
+
+import styles from "./SortableTableHeader.module.scss";
+
+interface SortableTableHeaderProps {
+  onClick: () => void;
+  wasActive: boolean;
+  lowToLarge: boolean;
```

Comment:
I noticed that these names came from the old component but they could be more clear:

`wasActive` => `isActive`
`lowToLarge` => `isAscending`

Reply:
Agree, need to rename this couple